.env.example

```diff
@@ -1,12 +1,12 @@
-DEV_MODE=false
+DEV_MODE=true
 ELASTICSEARCH_HOST=http://elasticsearch:9200
 REDIS_HOST=redis
 REDIS_PORT=6379
 APP_ENV=prod
```

Comment:
Everything else is dev, why not APP_ENV?

Reply:
just so that people configuring their .env know what the options are.

Reply:
do you want this to be changed to dev, too?

Reply:
Just think it makes more sense to be content.  If we want them to know their options (without looking at the readme), add comments explaining the options.

Reply:
@alamais-df please check this change again.